alpine/Dockerfile

```diff
@@ -24,18 +24,17 @@ ENV \
     USER_ID="9001"
 
 # Basic build-time metadata as defined at http://label-schema.org
```

Comment:
Thanks for updating the labels!
Can you also update the `http://label-schema.org` URLs in the comments?

Reply:
Done :)